### DISCUSSION TITLE
client for one of the services

### DISCUSSION THREAD

**Discussion by shreyas1599** [pd4d10/git-touch#176] [Ideas]
I want to write a client wrapper for one of the services (maybe bitbucket / gitea, not sure). So far I have not written anything useful in dart from scratch, so I was thinking of doing this. Can I add a repository to the git-touch organization and work on it (I'll use [this](https://github.com/SpinlockLabs/github.dart) as inspiration)? Might take a couple of months before it is usable on the git-touch repo. If I'm able to do it for one service, then perhaps I'll go ahead and write for others as well. What do you think @pd4d10 ?

**Comment by pd4d10** [comment]
OK. Sounds good
